pkg/gardenlet/apis/config/v1alpha1/types.go

```diff
@@ -507,6 +510,35 @@ type LoadBalancerServiceConfig struct {
 	Annotations map[string]string `json:"annotations"`
 }
 
+// MonitoringConfig contains settings for the monitoring stack.
+type MonitoringConfig struct {
+	// Shoot is optional and contains settings for the shoot monitoring stack.
+	// +optional
+	Shoot *ShootMonitoringConfig `json:"shoot,omitempty"`
+}
+
+// ShootMonitoringConfig contains settings for the shoot monitoring stack.
+type ShootMonitoringConfig struct {
+	// RemoteWrite is optional and contains remote write setting.
+	// +optional
+	RemoteWrite RemoteWriteMonitoringConfig `json:"remoteWrite,omitempty"`
```

Comment:
If this is optional then it should be a pointer type, otherwise (if it's required) please remove `omitempty` and the `// +optional` tag